ga4gh/exceptions.py

```diff
@@ -0,0 +1,179 @@
+"""
+Exceptions for the GA4GH server. Each exception that can occur in the server
+is given a unique error code that is derived from its name.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import zlib
+
+import ga4gh.protocol as protocol
+
+
+def convertException(exception):
```

Comment:
I don't like the name of this method; it's ambiguous.  Convert exception to what?

maybe convertToServerError ?